PlayCoreKtx/.google/packaging.yaml

```diff
@@ -0,0 +1,27 @@
+# GOOGLE SAMPLE PACKAGING DATA
+#
+# This file is used by Google as part of our samples packaging process.
+# End users may safely ignore this file. It has no relevance to other systems.
+---
+status:       PUBLISHED
+technologies: [Android]
+categories:   [Dynamic Apps]
```

Comment:
"Dynamic Apps" is not something we use externally
or is it just a metadata name for some internal search and it's OK?